spec/api/made_to_measure/subscribers_spec.rb

```diff
@@ -8,6 +8,7 @@
     let(:email) { 'flap@flappington.com' }
     let(:subscribers_path) { '/api/v0.1/subscribers' }
     let(:subscriber_path) { "/api/v0.1/subscribers/#{subscriber.id}" }
+    let(:subscriber_data_path) { "#{subscriber_path}/data" }
```

Comment:
Agreed.  This may be something to look into when we move into MVP if we change languages.